src/components/ButtonGroup.tsx

```diff
@@ -29,7 +29,7 @@ export function ButtonGroup(props: ButtonGroupProps) {
   const { buttons, disabled = false, size = "sm" } = props;
   const tid = useTestIds(props, "buttonGroup");
   return (
-    <div {...tid} css={sizeStyles[size]}>
+    <div {...tid} css={{ ...Css.df.$, ...sizeStyles[size] }}>
```

Comment:
Suggestion: I _think_ this could be `css={Css.df.add(sizeStyles[size]).$}`. Not sure.